aws-android-sdk-iot/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -767,6 +770,14 @@ public void onStatusChanged(AWSIotMqttClientStatus status, Throwable throwable)
                     statuses.add(status);
                 }
             });
+        } else if (AWSIotMqttManager.AuthenticationMode.USERNAME_PASSWORD.equals(authMode)) {
```

Comment:
Now that this control structure has moved from a ~simple `if/if-else`, you should probably move to a `switch` on `authMode`.